src/lib.rs

```diff
@@ -263,6 +263,7 @@ impl XSave {
     #[inline]
     #[cfg(feature = "asm")]
```

Comment:
Are these `cfg(feature = "asm")` still necessary?

Reply:
@bstrie  nightly versus stable still

Reply:
Ah, by making this work on stable you mean make this work on what is stable today, not make it work on the version of Rust that will be stable in two weeks, when `asm` is stabilized.